sphinx_automodapi/autodoc_enhancements.py

```diff
@@ -58,7 +60,19 @@ def type_object_attrgetter(obj, attr, *defargs):
                 return base.__dict__[attr]
             break
 
-    return getattr(obj, attr, *defargs)
+    try:
+        return getattr(obj, attr, *defargs)
+    except AttributeError as e:
```

Comment:
extreme nitpick: it's best to avoid single-letter variables and use something descriptive, even when the single letter feels trivial.